mailpoet/lib/Automation/Integrations/MailPoet/Templates/Templates.php

```diff
@@ -0,0 +1,71 @@
+<?php declare(strict_types=1);
+
+namespace MailPoet\Automation\Integrations\MailPoet\Templates;
+
+use MailPoet\Automation\Engine\Workflows\Step;
+use MailPoet\Automation\Engine\Workflows\Workflow;
+use MailPoet\Automation\Integrations\Core\Actions\WaitAction;
+use MailPoet\Automation\Integrations\MailPoet\Actions\SendWelcomeEmailAction;
+use MailPoet\Automation\Integrations\MailPoet\Triggers\SegmentSubscribedTrigger;
+
+class Templates {
+
+  /** @var WaitAction */
+  private $waitAction;
+
+  /** @var SegmentSubscribedTrigger */
+  private $segmentSubscribedTrigger;
+
+  /** @var SendWelcomeEmailAction */
+  private $sendWelcomeEmailAction;
+
+  public function __construct(
+    SegmentSubscribedTrigger $segmentSubscribedTrigger,
+    SendWelcomeEmailAction $sendWelcomeEmailAction,
+    WaitAction $waitAction
+  ) {
+    $this->waitAction = $waitAction;
+    $this->segmentSubscribedTrigger = $segmentSubscribedTrigger;
+    $this->sendWelcomeEmailAction = $sendWelcomeEmailAction;
+  }
+
+  public function delayedEmailAfterSignup(string $name): Workflow {
+    $triggerStep = $this->segmentSubscribedTriggerStep();
+
+    $waitStep = $this->waitStep(60 * 60);
+    $triggerStep->setNextStepId($waitStep->getId());
+
+    $sendEmailStep = $this->sendEmailActionStep();
+    $waitStep->setNextStepId($sendEmailStep->getId());
+    
+    $steps = [
+      $triggerStep,
+      $waitStep,
+      $sendEmailStep,
+    ];
+
+    return new Workflow($name, $steps);
+  }
+
+  private function waitStep(int $seconds): Step {
+    return new Step($this->uniqueId(), Step::TYPE_ACTION, $this->waitAction->getKey(), null, [
+      'seconds' => $seconds,
+    ]);
+  }
+
+  private function segmentSubscribedTriggerStep(?int $segmentId = null): Step {
+    return new Step($this->uniqueId(), Step::TYPE_TRIGGER, $this->segmentSubscribedTrigger->getKey(), null, [
+      'segment_id' => $segmentId,
+    ]);
+  }
+
+  private function sendEmailActionStep(?int $newsletterId = null): Step {
+    return new Step($this->uniqueId(), Step::TYPE_ACTION, $this->sendWelcomeEmailAction->getKey(), null, [
+      'welcomeEmailId' => $newsletterId
+    ]);
+  }
+
+  private function uniqueId(): string {
+    return bin2hex(random_bytes(7));
+  }
+}
```

Comment:
Oof, I thought I caught these but apparently missed this one. 

I didn't have EditorConfig support enabled in PhpStorm, so it should be working for me now. I know I previously had this other setting enabled, but somehow it got turned off:
<img width="323" alt="image" src="https://user-images.githubusercontent.com/8656640/172194600-0c4eeb65-db9b-43f7-b855-3e0b61e6b646.png">

Thanks for catching this!

Reply:
Cool, Jan L. did a [great improvement of the .editorconfig](https://github.com/mailpoet/mailpoet/compare/trunk...editorconfig) as his meetup hackaton topic so soon it may become even more useful in PHPStorm.